hack/lib/test.sh

```diff
@@ -328,12 +328,16 @@ kube::test::if_has_string() {
   local match=$2
 
   if grep -q "${match}" <<< "${message}"; then
+    echo -n "${green}"
```

Comment:
When running the version tests, I noticed that some of the helpers it uses had nice colorization and others did not. This is in a separate commit, and I can easily remove it from the PR if it is unwanted.